k8s/resources/study/dynamic-simulation-server-deployment.yaml

```diff
@@ -28,8 +28,6 @@ spec:
               name: dynawo-config-configmap-volume
             - mountPath: /home/powsybl/.itools
               name: dynawo-itools-configmap-volume
-            - mountPath: /home/powsybl/.itools/tmp
-              name: itoolstmp-emptydir
```

Comment:
files model.par, network.par and solver.par are no longer exported in tmp to interface to dynawaltz, we can remove this mount